dockerfiles/Dockerfile.cpu

```diff
@@ -0,0 +1,23 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+# Official user quickstart docker container for ONNX Runtime
+# Ubuntu 16.04, CPU version, Python 3.
+#--------------------------------------------------------------------------
+
+FROM ubuntu:16.04
```

Comment:
is it the conda python distro?

Reply:
I used the pip onnx / onnxruntime distribution!

```
RUN pip3 install numpy onnx
RUN pip3 install onnxruntime
```